spot-oa/ui/ipython/custom/custom.css

```diff
@@ -65,81 +65,81 @@
     background-image: url(./ajax-loader.gif);
 }
 
-.oni.oni_easy_mode:not(.oni_easy_mode_loading) #oni_easy_mode_loader
+.spot.spot_easy_mode:not(.spot_easy_mode_loading) #spot_easy_mode_loader
 {
     right: 25px;
     margin-top: 0px;
     width: auto;
 }
 
-.oni.oni_ninja_mode #oni_easy_mode_loader
+.spot.spot_ninja_mode #spot_easy_mode_loader
```

Comment:
Sure, we can discuss that. I used ninja as in "CSS ninja" or "JavaScript ninja", those terms refer to "advanced" knowledge/usage of a technology. Thanks